api/envoy/admin/v2alpha/server_info.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package envoy.admin.v2alpha;
+
+import "google/protobuf/duration.proto";
+
+// [#protodoc-title: Server State]
+
+// Proto representation of the value returned by /server_info, containing
+// server version/server status information.
+message ServerInfo {
+  // Server version.
+  string version = 1;
```

Comment:
One thing we could think about doing is adding more structure to the version, which actually has sub-components; e.g. the git SHA, the type of build, etc. This is just a raw string at https://github.com/envoyproxy/envoy/blob/master/api/envoy/api/v2/core/base.proto#L74 as well, so I think it's fine in this PR, but I think it would be nice to do a better job at this later on.